File: laramie-importacion-webapi/src/domain/entities/persona-juridica.ts
export default class PersonaJuridica {

    id: number;
	idTipoDocumento: number;
	numeroDocumento: string;
	denominacion: string;
	nombreFantasia: string;
	idFormaJuridica: number;
	idJurisdiccion: number;
	fechaConstitucion: Date;
	mesCierre: number;
	logo: string;

	constructor(
        id: number = 0,
		idTipoDocumento: number = 0,
		numeroDocumento: string = "",
		denominacion: string = "",
		nombreFantasia: string = "",
		idFormaJuridica: number = 0,
		idJurisdiccion: number = 0,
		fechaConstitucion: Date = null,
		mesCierre: number = 0,
		logo: string = ""
	)
	{
        this.id = id;
		this.idTipoDocumento = idTipoDocumento;
		this.numeroDocumento = numeroDocumento;
		this.denominacion = denominacion;
		this.nombreFantasia = nombreFantasia;
		this.idFormaJuridica = idFormaJuridica;
		this.idJurisdiccion = idJurisdiccion;
		this.fechaConstitucion = fechaConstitucion;
		this.mesCierre = mesCierre;
		this.logo = logo;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idTipoDocumento = row.idTipoDocumento ?? 0;
		this.numeroDocumento = row.numeroDocumento ?? "";
		this.denominacion = row.denominacion ?? "";
		this.nombreFantasia = row.nombreFantasia ?? "";
		this.idFormaJuridica = row.idFormaJuridica ?? 0;
		this.idJurisdiccion = row.idJurisdiccion ?? 0;
		this.fechaConstitucion = row.fechaConstitucion ?? null;
		this.mesCierre = row.mesCierre ?? 0;
		this.logo = row.logo ?? "";
	}

}

File: modules/administracion/models/persona_juridica.py
from sqlalchemy import Column, BigInteger, String, Integer, Date, Boolean

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class PersonaJuridica(Base):
    __tablename__ = "administracion_personas_juridicas"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_tipo_documento = Column(BigInteger, nullable=False)
    numero_documento = Column(String(50), nullable=False)
    denominacion = Column(String(250), nullable=False)
    nombre_fantasia = Column(String(250), nullable=True)
    id_forma_juridica = Column(BigInteger, nullable=True)
    fecha_constitucion = Column(Date, nullable=True)
    mes_cierre = Column(Integer, nullable=True)
    activo = Column(Boolean, nullable=False, default=True)
